src/components/DraggableLineChart/DraggableLineChartD3.ts

```diff
@@ -301,6 +311,29 @@ class DraggableLineChartD3 {
 					.y((d: any) => this.yScale(d.y))
 			);
 	};
+	renderEmptyRenderProp = (height: number, width: number) => {
+		const { emptyRenderProp } = this.params;
+		if (!emptyRenderProp || !this.shouldShowPreselect()) {
+			return;
+		}
+
+		const emptyDataObject = this.selection.selectAll('.emptyRender');
+		console.log({ emptyDataObject });
+		if (emptyDataObject.empty()) {
+			const emptyRender: ISelection = this.selection
+				.selectAll('.overlayContainer')
+				.append('foreignObject')
+				.attr('height', height)
+				.attr('width', width)
+				.attr('x', 80)
```

Comment:
@celinrayen do you know if this is consistent? It appears to be just given static value with the charts.

Reply:
I believe it should be `margin.left`